src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,68 @@
-export const Pagination = () => {};
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: CallableFunction,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const lastPageId = Math.ceil(total / perPage);
+  const handlePageChange = (step:number) => {
+    onPageChange((prevPage: number) => prevPage + step);
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled="true"
+          onClick={() => {
+            handlePageChange(-1);
+          }}
+        >
+          «
+        </a>
+      </li>
+      {getNumbers(1, lastPageId).map(number => (
+        <li
+          className={`page-item ${number === currentPage ? 'active' : ''}`}
+          key={`elem${number}`}
+        >
+          <a
+            href={`#${number}`}
+            data-cy="pageLink"
+            className="page-link"
+            onClick={() => {
+              handlePageChange(+number - currentPage);
+            }}
+          >
+            {number}
+          </a>
+        </li>
+      ))}
+      <li className={`page-item ${currentPage === lastPageId ? 'disabled' : ''}`}>
+        <a
+          data-cy="nextLink"
+          className="page-link"
+          href="#next"
+          aria-disabled="false"
```

Comment:
1. `«` and `»` links should open the prev and the next pages accordingly
    - disable each of them if it is already the first or the last page (use `li.disabled` and `a[aria-disabled="true"]`)